### PR TITLE
Cuav v5nano

### PR DESCRIPTION
**Pull request by auturgy**
Add cuav autopilot images and cuav v5 nano overview.  I've intentionally used the dual language images, but can change to english-only if others prefer

### CODE REVIEW

File: common/source/docs/common-cuav-v5nano-overview.rst [hunk 1]
```diff
@@ -0,0 +1,115 @@
+.. _common-cuav-v5nano-overview:
+
+============================
+CUAV V5 Nano Overview
+============================
```
Comment (khancyr) [inline common/source/docs/common-cuav-v5nano-overview.rst]: too long

Comment (auturgy) [reply common/source/docs/common-cuav-v5nano-overview.rst]: fixed

**Comment by rmackay9** [comment]
Hope I didn't merge too early but they are new pages and sometimes it's just easier to review on the real web page.

File: common/source/docs/common-autopilots.rst [hunk 2]
```diff
@@ -16,6 +16,7 @@ Open hardware
     The Cube <common-thecube-overview>
     Pixracer <common-pixracer-overview>
     CUAV v5 <common-pixhackV5-overview>
+    CUAV v5 Nano <common-cuav-v5nano-overview>
```
Comment (tridge) [inline common/source/docs/common-autopilots.rst]: missing a link to v5+ plage?

Comment (auturgy) [reply common/source/docs/common-autopilots.rst]: Yes. I don’t know whether to deprecate v5 and just have v5-plus, or have both.
Probably no harm having both until I find out.

**Comment by auturgy** [comment]
I’ve had a chat to hongle and merged a fix

**Pull request merged**